JSDemos/Demos/Common/LabelMode/Angular/app/app.component.html

```diff
@@ -0,0 +1,166 @@
+<div class="title">Edit Employee Info</div>
+<div class="editors">
+  <div class="editors-container">
+    <div class="left">
```

Comment:
display:inline-block and width: 49% can cause screenshots difference in visual tests. Flex will be the right solution.